sycl/doc/extensions/experimental/sycl_ext_oneapi_bindless_images.asciidoc

```diff
@@ -1964,4 +1964,5 @@ These features still need to be handled:
 |4.3|2023-09-08| - Clarify how normalized image formats are read
                  - Remove support for packed normalized image formats 
                    (`unorm_short_555`, `unorm_short_565`, `unorm_int_101010`)
+|4.4|2023-09-12| - Update wording for allocating images + fix typo
```

Comment:
This will clash with the proposal changes in https://github.com/intel/llvm/pull/11151

https://github.com/intel/llvm/pull/11151/files#diff-5032f8f6324e64548fb593ae0d33a4a1fa357a89e77467d04a8a77208e28a556R1977

Reply:
#11151 has been merged. I've resolved the conflict. This can now be merged.